integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/model/Envelope.java

```diff
@@ -0,0 +1,32 @@
+package org.symphonyoss.integration.provisioning.client.model;
+
+/**
+ * Wrap the data into an envelope.
```

Comment:
Add information here to explain that the Apps Repository API wraps all request and response content in a "data" object and that is the purpose of this wrapper class.